charts/hedera-json-rpc-relay/values.yaml

```diff
@@ -142,15 +142,20 @@ redis:
   # REDIS_URL=redis://<release-name>-redis-master:6379
   autoconfig: true
 
-# Disabled newman temporarily
 cronjob:
   enabled: false
+  # Note: hoppscotch/hopp-cli does not exist as a standalone Docker image.
+  # The CLI is distributed only as an npm package (@hoppscotch/cli).
+  # We use the official hoppscotch/hoppscotch AIO image which includes Node.js,
+  # and install the CLI at runtime via npm.
   image:
     pullPolicy: IfNotPresent
-    repository: postman/newman
-    tag: 5.3.1-alpine
-  # Custom postman.json in base64 encoding
-  postman: ''
+    repository: hoppscotch/hoppscotch
+    tag: 2025.11.1
```

Comment:
How do we now its co-related to 0.30.0. Shall we add a comment?

Reply:
thanks for the clarification. That said npm install is not pinned to 0.30.0.
As we want to avoid any unnecsary dependencies.  we should make the version static. with @0.30.0

Aside this why do we need this base image when on our package json we only need the cli?